app/components/agenda/agenda-header/agenda-version-actions.js

```diff
@@ -333,6 +333,16 @@ export default class AgendaAgendaHeaderAgendaVersionActions extends Component {
       this.showNotAllowedToast();
       return;
     }
+    const hasJobAfterOpenModal = !!(await this.store.queryOne('job', {
+      'filter[status]': CONSTANTS.JOB_STATUSSES.SUCCESS,
+      'filter[:gte:time-ended]': this.openAgendaCheckTimestamp.toISOString(),
+    }));
```

Comment:
Job is a very widely used type. any job from any service will be queried here.
We need to specify `DocumentNamingJob`